manager/manager_test.go

```diff
@@ -30,17 +30,19 @@ func newService() manager.Service {
 func TestRegister(t *testing.T) {
 	svc := newService()
 
-	cases := map[string]struct {
+	type testCase struct {
+		desc string
 		user manager.User
 		err  error
-	}{
-		"register new user":      {user, nil},
-		"register existing user": {user, manager.ErrConflict},
+	}
+	cases := []testCase{
```

Comment:
For the sake of consistency, use the sort approach presented in https://nathanleclaire.com/blog/2014/04/27/a-surprising-feature-of-golang-that-colored-me-impressed/.